src/mono/mono.proj

```diff
@@ -655,6 +655,35 @@
 
   <Import Project="Directory.Build.targets" />
 
+  <Target Name="BuildAppleAppBuilder">
+    <MSBuild Projects="$(MonoProjectRoot)AppleAppBuilder\AppleAppBuilder.csproj"
+             Properties="Configuration=$(Configuration)"
+             Targets="Restore;Build" />
+  </Target>
+
+  <UsingTask TaskName="AppleAppBuilderTask" 
+             AssemblyFile="$(ArtifactsObjDir)mono\AppleAppBuilder\$(TargetArchitecture)\$(Configuration)\AppleAppBuilder.dll" />
+
+  <Target Name="BuildAppleApp" DependsOnTargets="BuildAppleAppBuilder">
+    <AppleAppBuilderTask 
+        Arch="$(TargetArchitecture)"
+        ProjectName="$(ProjectName)"
+        Optimized="$(Optimized)"
+        MonoInclude="$(BinDir)include\mono-2.0"
```

Comment:
This should better come from runtime pack

Reply:
cc @steveisok @akoeplinger 
how do I build the "runtime pack", I've just built my repo with `build.sh -c Release -os iOS -arch arm64 -subset Mono+Libs+Libs.Tests` and don't see any runtime packs. The only location where I can find `mono-2.0` headers and `mono-aot-cross` is this `$(BinDir)`

Reply:
It's in a [PR](https://github.com/dotnet/runtime/pull/34662).